test/verifyMembership.js

```diff
@@ -0,0 +1,57 @@
+const Molecule = artifacts.require("Molecule");
+// const CkbLightClient = artifacts.require('CkbLightClient');
+const CkbLightClientMock = artifacts.require('CkbLightClientMock');
+const CkbMbt = artifacts.require('CkbMbt');
+const Blake2b = artifacts.require('Blake2b');
+const CkbProof = artifacts.require("CkbProof");
+
+const fs = require('fs');
+const proof_path = require('path');
+const ethers = require('ethers');
+
+contract("CkbProof", (accounts) => {
+  it("test verifyMembership", async () => {
+    const molecule = await Molecule.new();
+    console.log("molecule deployed on ", molecule.address);
+    await CkbProof.link(molecule.address);
+
+    const ckbLightClient = await CkbLightClientMock.new();
+    console.log("ckbLightClient deployed on ", ckbLightClient.address);  
+    let blockNumberHexString = "0x2a";
+    let blockNumber = ethers.utils.hexZeroPad(blockNumberHexString, 32);
+    const header = await ckbLightClient.getHeader(blockNumber);
+    console.log("header transactionsRoot", header.transactionsRoot);  
+
+    await CkbProof.link(ckbLightClient.address);
+
+    const ckbMbt = await CkbMbt.new();
+    console.log("ckbMbt deployed on ", ckbMbt.address);    
+    await CkbProof.link(ckbMbt.address);
+
+    const blake2b = await Blake2b.new();
+    await CkbProof.link(blake2b.address);
+    console.log("blake2b deployed on ", blake2b.address);
+
+    const ckbProofInstance = await CkbProof.new();
```

Comment:
`CkbProof`` is a library, not a contract, and therefore cannot be deployed using new.

See https://trufflesuite.com/docs/truffle/how-to/contracts/run-migrations/#deployerlinklibrary-destinations